examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -244,6 +246,22 @@ CHIP_ERROR InitCommissionableDataProvider(LinuxCommissionableDataProvider & prov
                          options.payload.discriminator.GetLongValue());
 }
 
+void PrintEndpointsVendorAndProductIDs(
+    const std::vector<matter::casting::memory::Strong<matter::casting::core::Endpoint>> & endpoints)
+{
+    ChipLogProgress(AppServer, "simple-app-helper.cpp::PrintEndpointsVendorAndProductIDs() Number of Endpoints: %d",
+                    static_cast<int>(endpoints.size()));
+    for (const auto & endpoint : endpoints)
+    {
+        int endpointId = endpoint->GetId();
+        int vendorId   = endpoint->GetVendorId();
+        int productId  = endpoint->GetProductId();
+        ChipLogProgress(AppServer,
+                        "simple-app-helper.cpp::PrintEndpointsVendorAndProductIDs() Endpoint ID: %d, Vendor ID: %d, Product ID: %d",
+                        endpointId, vendorId, productId);
```

Comment:
Add a LogDetail() function in the Endpoint class that you can just call from here.

Reply:
sure.